site/content/en/docs/manual/basics/types-of-shapes.md

```diff
@@ -10,6 +10,7 @@ There are five shapes which you can annotate your images with:
 - `Polygon`
 - `Polyline`
 - `Points`
+- `Ellipse`
```

Comment:
Only one comment, line above is:
``There are five shapes which you can annotate your images with:``

But this list has 8 annotation types